taf/repositoriesdb.py

```diff
@@ -94,7 +98,10 @@ def load_repositories(
             repositories = _get_json_file(auth_repo, repositories_path, commit)
             targets = _get_json_file(auth_repo, targets_path, commit)
         except InvalidOrMissingMetadataError as e:
-            taf_logger.warning("Skipping commit {} due to error {}", commit, e)
+            log_func = taf_logger.warning
```

Comment:
Could we just log with _debug_ level? Not sure what benefits we have with separating these logs, but I would rather raise some `RepositoriesJsonMissingError` exception and catch it here if needed.